test/aircraft/Pilot/descendViaStar.spec.js

```diff
@@ -1,43 +1,75 @@
 import ava from 'ava';
-
 import Pilot from '../../../src/assets/scripts/client/aircraft/Pilot/Pilot';
 import {
-    fmsArrivalFixture,
-    modeControllerFixture
+    createFmsArrivalFixture,
+    createModeControllerFixture
 } from '../../fixtures/aircraftFixtures';
 
+// Fixtures
+let pilot = null;
 
-ava('.descendViaStar() sets the correct Mcp modes and values', (t) => {
-    const pilot = new Pilot(modeControllerFixture, fmsArrivalFixture);
+// Mocks
+const successResponse = [true, 'descend via STAR'];
+const failureResponse = [false, 'unable to descend via STAR'];
```

Comment:
these two vars should include `mock` in their name

Reply:
done